_data/sites.json

```
@@ -12114,6 +12114,16 @@
         ]
     },
 
+    {
+        "name": "NationStates",
+        "url": "https://www.nationstates.net/page=faq",
+        "difficulty": "medium",
+        "notes": "Stay logged out of your nation for 28 or 60 days, depending on your settings.",
+        "domains": [
+            "nationstates.net"
```

Add the domain for the forum:

```suggestion
            "nationstates.net",
            "forum.nationstates.net"
```